molecule/default/tests/test_default.py

```diff
@@ -42,4 +42,8 @@ def test_packages(host, pkg):
 @pytest.mark.parametrize("pkg", ["gophish-init"])
 def test_pip_packages(host, pkg):
     """Test that the pip packages were installed."""
-    assert pkg in host.pip.get_packages(pip_path="pip3")
+    # Note that we are using the version of pip3 in the Python virtual
```

Comment:
Not sure if it makes any difference since pip is pip3 here, so feel free to ignore:
```suggestion
    # Note that we are using the version of pip in the Python virtual
```